src/modules/locale/LocaleConfiguration.cpp

```diff
@@ -201,6 +201,8 @@ LocaleConfiguration::fromLanguageAndLocation( const QString& languageLocale,
                 // but nearly all its native speakers also speak English,
                 // and migrants are likely to use English.
                 { "IE", "en" },
+                //India has many languages even though hindi is known as national language but english is used in all computer,mobile devices.
+                { "IN", "en" },
```

Comment:
Is this needed? 0e7982ec371246cc6e189ada32b2787f204479e5 already switched off Hindi-for-India AFAIK.

Reply:
(and if it is needed -- you can test more easily than I can because I need to spoof locations -- then the comment should say "Hindi" and "English" etc.